File: 250724/Next Level/next-level.java
import java.util.*;


class User {

    String userId;
    int level;


    public User() {
        this.userId = "codetree";
        this.level = 10;
    }

    public User(String userId, int level) { 
        this.userId = userId;
        this.level = level;
    }
}

public class Main {
    public static void main(String[] args) {
        Scanner sc = new Scanner(System.in);
        String userId = sc.next();
        int level = sc.nextInt();

        User user1 = new User();

        System.out.println("user " + user1.userId + " lv " + user1.level);
        
        User user2 = new User(userId, level);

        System.out.println("user " + user2.userId + " lv " + user2.level);


    }
}

File: 250724/Next Level/next-level.py
class User :
    def __init__ (self, user_Id = "codetree", level=10):
        self.u = user_Id;
        self.l = level;


user_id, level = input().split();

level = int(level);

user1 = User();

print(f"user {user1.u} lv {user1.l}")

user2 = User(user_id, level)

print(f"user {user2.u} lv {user2.l}")

